### DISCUSSION TITLE
Lightroom Classic 2024v13 and MIDI controller Behringer BCR2000

### DISCUSSION THREAD

**Discussion by PaulHellemans** [rsjaffe/MIDI2LR#767] [Problems]
I am now using Lightroom classic version 2024v13.0.0.15 and MIDI2LR-6.0.1.0 Windows. After starting Ligtroom everything works for 1 to 2 minutes and then communication between Lightroom and MIDI2LR disappears. I can then only continue working with the PC keyboard. I use the Behringer BCR2000 as a MIDI controller. Does anyone also have this problem and does anyone know a solution? In previous versions I didn't have this problem. Many thanks.

**Comment by antzpantz** [comment]
Have you tried uninstalling (delete the directory
C:\Users\Administrator\Midi2LR\MIDI2LR.lrplugin) and reinstall it. and
reinstalling MIDI2LR?

Or are there other MIDI devices (virtual or physical) you have plugged in?
StreamDecks or other devices?




On Wed, 20 Dec 2023 at 21:55, PaulHellemans ***@***.***>
wrote:
>
> I am now using Lightroom classic version 2024v13.0.0.15 and
MIDI2LR-6.0.1.0 Windows. After starting Ligtroom everything works for 1 to
2 minutes and then communication between Lightroom and MIDI2LR disappears.
I can then only continue working with the PC keyboard. I use the Behringer
BCR2000 as a MIDI controller. Does anyone also have this problem and does
anyone know a solution? In previous versions I didn't have this problem.
Many thanks.

**Reply by PaulHellemans** [reply]
I tried to completely uninstall and reinstall MIDI2LR.Ir Plugin. But this didn't help.
And....Yes, there is 'VirtualMIDISynth' from CoolSoft. This is a virtual midid controller for JJazzLab (a music program). Could this be the problem?

**Comment by antzpantz** [comment]
Probably. Try running Lightroom and MIDI2LR with ONLY the BCR connected and
no other devices (virtual or physical) connected. The issue needs to be
isolated and more details are needed, like the MIDI2LR log file

**Comment by PaulHellemans** [comment]
I removed VirtualMIDISynth' from CoolSoft from my computer and checked everything and there are no other MIDI controllers present. The problem has not been solved. Only the command 'Previus photo' and 'Next photo' work, that's all.

**Comment by antzpantz** [comment]
Have you tried another USB cable?

**Comment by PaulHellemans** [comment]
Yes, It makes no difference.
